content/docs/migration_1.0/Concepts.md

```diff
@@ -0,0 +1,126 @@
+---
+title: "Concepts"
+weight : 6100
+menu:
+  docs:
+    parent: migration_1.0
+---
+
+The Zenoh team have been hard at work preparing an official version 1.0.0 of Zenoh!   
+This major release comes with several API changes, quality of life improvements and developer conveniences.
+
+We now have a more stable API and intend to keep backward compatibility in the future Zenoh revisions.  
+This guide is here to ease the transition to Zenoh 1.0.0 for our users!
+
+## Value is gone, long live ZBytes 
+We have replaced `Value` with `ZBytes` and `Encoding`.  
+`Zbytes` is the type core to data representation in Zenoh, all API's have be reworked to accept `ZBytes` or something that can be converted into a `ZBytes`.  
+We have added a number of conversion implementations for language primitives as well as methods to seamlessly allow user defined structs to be serialized into `ZBytes`.  
+`Sample`'s payloads are now `ZBytes`, `Publishers`, `Queryables` and `Subscribers` now expect `ZBytes` for all their interfaces.
```

Comment:
`Attachment` as well. Add the mention here and add a link to the dedicated section.